lib/Backends/NNPI/NNPICompiledFunction.cpp

```diff
@@ -723,6 +727,12 @@ static const std::string tensorToJSON(const NNPICompiledTensor &tensor) {
   fs << "\"type\" : \"" << tensor.type << "\"," << std::endl;
   fs << "\"alloc\" : \"" << dumpAllocType(tensor.allocType) << "\","
      << std::endl;
+  fs << "\"possible_alloc\" : ";
+  for (auto it = tensor.possibleAlloc.begin(); it != tensor.possibleAlloc.end();
+       it++) {
+    fs << "\"" << dumpAllocType(*it) << "\" ";
```

Comment:
If this is tensorTOJSON, then how could 
```
"possible_alloc": "foo" "bar" 
```
be converted to JSON?
It should be something like
```
"possible_alloc": ["foo", "bar"],
```
right?

Reply:
Thanks, will fix it.